llama_index/indices/struct_store/sql_query.py

```diff
@@ -203,6 +206,150 @@ async def _aquery(self, query_bundle: QueryBundle) -> Response:
         return response
 
 
+class SQLTableQueryEngine(BaseQueryEngine):
+    def __init__(
+        self,
+        sql_database: SQLDatabase,
+        **kwargs: Any,
+    ) -> None:
+        self._sql_database = sql_database
+        super().__init__(**kwargs)
+
+    def _query(self, query_bundle: QueryBundle) -> Response:
+        """Answer a query."""
+        # NOTE: override query method in order to fetch the right results.
+        # NOTE: since the query_str is a SQL query, it doesn't make sense
+        # to use ResponseBuilder anywhere.
+        response_str, extra_info = self._sql_database.run_sql(query_bundle.query_str)
+        response = Response(response=response_str, extra_info=extra_info)
+        return response
+
+    async def _aquery(self, query_bundle: QueryBundle) -> Response:
+        return self._query(query_bundle)
```

Comment:
Hmm so for this module we don't even use the LLM at all? Neither text to sql query, nor sql response to text?

Reply:
yeah i think i mentioned this earlier cc @hongyishi but we probably don't need this (this wrapper around the raw sql_database) doesn't really do much